modules/ssh-agent-using.adoc

```diff
@@ -191,14 +191,13 @@ endif::openshift-origin[]
 $ ssh-keygen -t ed25519 -N '' -f <path>/<file_name> <1>
 ----
 <1> Specify the path and file name, such as `~/.ssh/id_ed25519`, of the new SSH key. If you have an existing key pair, ensure your public key is in the your `~/.ssh` directory.
-+
-ifndef::ibm-power-vs[]
-[NOTE]
-====
-If you plan to install an {product-title} cluster that uses FIPS Validated / Modules in Process cryptographic libraries on the `x86_64` architecture, do not create a key that uses the `ed25519` algorithm. Instead, create a key that uses the `rsa` or `ecdsa` algorithm.
-====
-endif::ibm-power-vs[]
-
+//+
+//ifndef::ibm-power-vs[]
+//[NOTE]
+//====
+//If you plan to install an {product-title} cluster that uses FIPS Validated / Modules in Process cryptographic //libraries on the `x86_64` architecture, do not create a key that uses the `ed25519` algorithm. Instead, create a key that uses the `rsa` or `ecdsa` algorithm.
```

Comment:
What is the _//libraries_?

Reply:
Probably bad wrapping on my side. I will remove the slashes from libraries.